src/main/java/htsjdk/variant/variantcontext/writer/VCFWriter.java

```diff
@@ -125,6 +123,16 @@ public VCFWriter(final Path location, final OutputStream output, final SAMSequen
         this.writeFullFormatField = writeFullFormatField;
     }
 
+    /**
+     * See {@link VCFVersionUpgradePolicy}. Controls how the writer will handle headers and variant contexts
+     * from versions of VCF before the current version. The writer will call {@link VCFHeader#upgradeVersion} headers
+     * passed in by {@link VCFWriter#writeHeader} and {@link VCFWriter#setHeader} before writing the header out.
+     * @param policy the policy to use
+     */
```

Comment:
The javadoc in `VCFVersionUpgradePolicy` and `VariantContextWriterBuilder` only mentions `VCFWriter` but I added a little clarifying note that it has no effect on `BCF2Writer` in `VCFVersionUpgradePolicy`, `VariantContextWriterBuilder` and this method.

It doesn't affect BCF because BCF2Writer calls the static method which writes the header verbatim.